packages/cli/src/commands/stream/create.ts

```diff
@@ -0,0 +1,108 @@
+import { flags } from '@oclif/command'
+import { TileDocument } from '@ceramicnetwork/stream-tile'
+
+import { Command } from '../../command'
+import type { CommandFlags } from '../../command'
+import { parseContent, parseControllers } from '../../utils'
+
+type Flags = CommandFlags & {
+  determinitic?: boolean
+  'only-genesis'?: boolean
+
+  did?: string
+  controllers?: string
+}
+
+export default class Create extends Command<
+  Flags,
+  {
+    schema: string
+    content: string
+  }
+> {
+  static description = 'Create a new Stream'
+
+  static args = [
+    {
+      name: 'schema',
+      description: 'the StreamId of the schema to use',
+      required: true,
+    },
+    {
+      name: 'content',
+      description: 'the Stream body',
+      required: true,
+    },
+  ]
+
+  static flags = {
+    ...Command.flags,
+    'only-genesis': flags.boolean({
+      char: 'g',
+      description: 'only generate genesis block',
+      default: false,
+    }),
+    deterministic: flags.boolean({
+      char: 'd',
+      description: 'generate deterministic stream',
+      default: false,
+    }),
+
+    did: flags.string({
+      exclusive: ['key'],
+      description: 'Creator DID',
+    }),
+    controllers: flags.string({
```

Comment:
Could this be change to a singular `controller` please? Ceramic doesn't support multiple controllers so it doesn't really make sense to have the CLI documented as if it does.